.github/workflows/precommit.yml

```diff
@@ -120,18 +120,18 @@ jobs:
           TOX_PARALLEL_NO_SPINNER: 1
         shell: bash
       - if: ${{ (matrix.python-version == '3.9' || matrix.python-version == '3.10') && contains('macos', matrix.os.download_name) }}
-        name: Run tests (including doctest tests)
-        run: python -m tox -e "py${PYTHON_VERSION/\./}-udf"
+        name: Run doctests
+        run: python -m tox -e "py${PYTHON_VERSION/\./}-doctest-notudf"
         env:
           PYTHON_VERSION: ${{ matrix.python-version }}
           cloud_provider: ${{ matrix.cloud-provider }}
           PYTEST_ADDOPTS: --color=yes --tb=short
           TOX_PARALLEL_NO_SPINNER: 1
           SNOWFLAKE_IS_PYTHON_RUNTIME_TEST: 1
```

Comment:
`SNOWFLAKE_IS_PYTHON_RUNTIME_TEST: 1` seems like the only difference between 3.8 and 3.9/3.10, there should be a better way of specifying this instead of having separate steps.

Reply:
`SNOWFLAKE_IS_PYTHON_RUNTIME_TEST: 1` is used to uncap upper bound of Python version for testing when we add client support for a new Python version with no server-side support yet. Since Python3.9/3.10 are fully supported now, I'm removing the extra steps and adding a comment.